content/blog/mapr-for-predictive-maintenance.md

```diff
@@ -0,0 +1,318 @@
+---
+title: MapR for Predictive Maintenance
+date: 2022-02-15T05:31:46.014Z
+author: Ian Downard
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+---
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+This project is intended to show how to build Predictive Maintenance applications on MapR. Predictive Maintenance applications place high demands on data streaming, time-series data storage, and machine learning. Therefore, this project focuses on data ingest with MapR Streams, time-series data storage with MapR-DB and OpenTSDB, and feature engineering with MapR-DB and Apache Spark.
+
+## Overview:
+
+Predictive Maintenance requires a cutting edge data platform in order to handle fast streams of  IoT data, with the processing required for on-the-fly feature engineering, and the flexibility required for data science and machine learning.
```

Comment:
There is an extra space prior to "IoT data" that you can delete.
Change "in order to handle" to "that handles" and remove the comma after "data"  and then delete the comma after engineering so the sentence reads:
"Predictive Maintenance requires a cutting edge data platform that handles fast streams of IoT data with the processing required for on-the-fly feature engineering and the flexibility required for data science and machine learning."

Also, a quick note - in this first use of "IoT", might you want to spell it out as "Internet of Things (IoT)? That's the general practice. I do realize that most of this audience already understands IoT, so it might not be totally necessary.

Reply:
done